modules/test/playwright/pages/object-web/ModelBuilderPage.ts

```diff
@@ -218,6 +206,30 @@ export class ModelBuilderPage {
 		);
 	}
 
+	async openObjectFieldSelectionPage(
+		objectDefinitionName: string,
+		objectFieldBusinessType: string,
+		objectFieldLabel: string
+	) {
+		await this.leftSidebarItems
+			.filter({hasText: objectDefinitionName})
+			.click();
+
+		await this.objectDefinitionNodes
+			.filter({hasText: objectDefinitionName})
+			.getByRole('button', {name: 'Add Field or Relationship'})
+			.click();
+
+		await this.addObjectFieldButton.click();
+
+		await this.newObjectFieldLabel.fill(objectFieldLabel);
+
+		await this.newObjectFieldSelectBusinessType.click();
+		await this.page
+			.getByRole('option', {exact: true, name: objectFieldBusinessType})
+			.click();
+	}
```

Comment:
These three steps belong to the create `createObjectField()`. Please, be careful to not break other tests that rely on `ModelBuilderPage.ts`.